packages/nc-gui/components/project/spreadsheet/views/formView.vue

```diff
@@ -580,7 +580,7 @@ export default {
     },
     columns: {
       get() {
-        return this.fields.filter(f => f.show).sort((a, b) => a.order - b.order)
+        return this.fields.filter(f => f.show && f.uidt != UITypes.Rollup && f.uidt != UITypes.Lookup).sort((a, b) => a.order - b.order)
```

Comment:
`f.show && f.uidt != UITypes.Rollup && f.uidt != UITypes.Lookup`
this is not a debugable code.
if should be binded into a function
```
function someFuncName () {
   if(!f.show){
       console.log("reason 1")
       return false
   }
   if(f.uidt === UITypes.Rollup){
       console.log("reason 2")
       return false
   }
   if(f.uidt === UITypes.Lookup){
        console.log("reason 3")
        return false
   }
   return true
}
```

this way we will able to know easily what exactly happened,
again not mandatory, just a suggestion for good practice. :smiley: :smiley: